src/app/containers/CpsRecommendations/utils.js

```diff
@@ -1,3 +1,11 @@
+const getIchefSrc = href => {
+  const DEFAULT_IMAGE_RES = 660;
+
+  const path = href.replace('http://c.files.bbci.co.uk', '/cpsprodpb');
+
+  return `https://ichef.bbci.co.uk/news/${DEFAULT_IMAGE_RES}${path}`;
```

Comment:
Is this something that should be happening in Ares @jimjohnsonrollings ? Maybe we should raise it with them. If not arguably it should be in the pre-proc rules?

Reply:
I don't think this is an Ares thing.  Removing the original domain could be, but the ichef solution we use is specific to us and a different consumer of the API may choose to use a different ichef.

Reply:
we already have an image handler which constructs the ichef urls. could we not re-use that?

Reply:
I've switched to use the `StoryPromo` wrapper in simorgh, which does this out of the  box
https://github.com/bbc/simorgh/blob/93007af14b99f476605d3b8888e5127fd664b3d5/src/app/containers/StoryPromo/index.jsx#L46